### PR TITLE
MGMT-8666 - Fix onprem logs destination in prow

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @osherdp

### CODE REVIEW

File: scripts/download_logs.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ function download_service_logs() {
     podman ps -a || true
 
     for service in "assisted-service" "assisted-image-service" "assisted-installer-ui" "postgres"; do
-      podman logs ${service} >${LOGS_DEST}/onprem_${service}.log || true
+      podman logs ${service} >${LOGS_DEST}/k8s_${service}_onprem.log || true
```
Comment (osherdp) [inline scripts/download_logs.sh:24]: it will be misleading, because this deployment method has nothing to do with kubernetes, no?

**Review by osherdp — CHANGES_REQUESTED** [review]

File: scripts/download_logs.sh (hunk 1 above)
Comment (eliorerz) [reply scripts/download_logs.sh:24]: Right.. I think I didn't thought all the way through this.
I wanted to make the logs with the same prefix as the minikube deployments.
I'll change them both to start with a new prefix

File: scripts/download_logs.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ function download_service_logs() {
     podman ps -a || true
 
     for service in "assisted-service" "assisted-image-service" "assisted-installer-ui" "postgres"; do
-      podman logs ${service} >${LOGS_DEST}/onprem_${service}.log || true
+      podman logs ${service} >${LOGS_DEST}/logs${service}_${DEPLOY_TARGET}.log || true
```
Comment (osherdp) [inline scripts/download_logs.sh:24]:
```suggestion
      podman logs ${service} >${LOGS_DEST}/logs_${service}_${DEPLOY_TARGET}.log || true
```

Comment (osherdp) [reply scripts/download_logs.sh:24]: same for other stuff, just make sure it's displaying nicely

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**